yorkie/src/main/kotlin/dev/yorkie/core/Presences.kt

```diff
@@ -18,9 +18,9 @@ public class Presences private constructor(
 
     public operator fun minus(actorID: ActorID): Presences = (map - actorID).asPresences()
 
-    override fun hashCode(): Int = map.hashCode()
-
-    override fun equals(other: Any?): Boolean = map == (other as? Presences)?.map
```

Comment:
I removed it to ensure that `document.publishPresenceEvent()` is properly invoked whenever a new value is emitted to `document.presences`.